cukinia/common_security_tests.d/sudo.conf

```diff
@@ -44,8 +44,12 @@ CMD_WITHOUT_ARGS=$(
 as "SEAPATH-00154 - sudo commands always specify arguments" \
     cukinia_test -n $CMD_WITHOUT_ARGS
 
-as "SEAPATH-00155 - no command is specified with wildcard argument" \
-    not cukinia_cmd grep -q "^(?!#|Defaults).*=.*\* *$" $SUDOERS_FILES
+CMD_WILCARD=$(
+    grep -h '.*\*$' $SUDOERS_FILES |
+    grep -v -E '^ceph'
```

Comment:
It is possible to define for ceph without using wildcards. Use the Ansible ceph disk variable to know the disk.